tt

```diff
@@ -57,6 +55,10 @@ main() {
 		esac
 	fi
 
+	TT_PROJECTS=${TT_PROJECTS:-~/tmp}
+	TT_GIT_BASE=${TT_GIT_BASE:-git@github.com:meet-tarantino}
+	apply_workspace_vars # this may override the above defaults
```

Comment:
yeah, not sure if lines 58/59 need to exist, it was more for back-compat for my other workspace that uses those lol